app/domain/authentication/authn_jwt/signing_key/create_signing_key_provider.rb

```diff
@@ -54,13 +41,23 @@ def neither_sigining_keys_providers_configured
         def provider_uri_has_valid_configuration?
           return @provider_uri_has_valid_configuration if defined?(@provider_uri_has_valid_configuration)
 
-          @provider_uri_has_valid_configuration = fetch_provider_uri_signing_key.valid_configuration?
+          @provider_uri_has_valid_configuration = @check_authenticator_secret_exists.call(
+            conjur_account: @authentication_parameters.account,
+            authenticator_name: @authentication_parameters.authenticator_name,
+            service_id: @authentication_parameters.service_id,
+            var_name: PROVIDER_URI_RESOURCE_NAME
+          )
         end
 
         def jwks_uri_has_valid_configuration?
```

Comment:
what does it mean "has_valid_configuration"?
 the method only checks if it exists

Reply:
renamed to `jwks_uri_has_resource_exists?`